release_notes.md

```diff
@@ -2,6 +2,16 @@
 ### 0.3.0 _ under construction
   * [#3](https://github.com/ctaggart/froto/issues/3) F# type provider
   * [#15](https://github.com/ctaggart/froto/issues/15) froto.exe code generator for proto3
+  * [#37](https://github.com/ctaggart/froto/issues/38) Fix bug: O(n^2) Performance Problem on repeated fields
```

Comment:
#3 and #15 aren't actually done. We can make a 0.3.0 release and bump these to 0.4.0.